sdk/cosmos/spark.yml

```diff
@@ -52,3 +52,18 @@ stages:
       DatabricksToken: $(spark-databricks-token)
       SparkVersion: 'azure-cosmos-spark_3-3_2-12'
       ClusterName: 'oltp-ci-spark33-2workers-ds3v2'
+  - template: /sdk/cosmos/spark.databricks.yml
```

Comment:
FYI I think the inner yaml file could be changed to something like:

```
stages:
  - ${{ each version in parameters.SparkVersions }}
     < rest of yaml template indented one more level here >
     < reference individual spark version like ${{ version }} >
```

Then you could just have this at the `spark.yml` level:

```
variables:
  - template: ../../eng/pipelines/templates/variables/globals.yml
  - name: AdditionalArgs
    value: ''

stages:
  - template: /sdk/cosmos/spark.databricks.yml
    parameters:
      SparkVersions:
        - 'azure-cosmos-spark_3-3_2-12'
        - 'azure-cosmos-spark_3-4_2-12'
```

I'm happy to help get this set up if interested.